### PR TITLE
Fix: ng build --prod

### PR DESCRIPTION
**Pull request by chadtomkiss-madetech**
**Work done**

- Fixed visibility of component services used in templates to fix issue with AOT compilation
- Removed variables used in templates that do not exist

#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [ ] No, I found it difficult to test
- [x] No, they are not required for this change

### CODE REVIEW

File: src/app/features/notifications/notification-delink-to-parent/notification-delink-to-parent.component.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      <span class="govuk-caption-xl">{{ workplace?.name }}</span>
```
Comment (aaron-russell) [inline src/app/features/notifications/notification-delink-to-parent/notification-delink-to-parent.component.html:4]: Should this have been deleted?

File: src/app/shared/components/cancel-data-owner-dialog/cancel-data-owner-dialog.component.html [hunk 2]
```diff
@@ -1,17 +1,13 @@
 <app-error-summary *ngIf="isCancelOwnershipError" [serverError]="serverError" [form]="form"> </app-error-summary>
 
-<div class="govuk-form-group" [class.govuk-form-group--error]="submitted && form.invalid">
```
Comment (aaron-russell) [inline src/app/shared/components/cancel-data-owner-dialog/cancel-data-owner-dialog.component.html:3]: Should this have been deleted?

File: src/app/features/notifications/notification-delink-to-parent/notification-delink-to-parent.component.html (hunk 1 above)
Comment (chadtomkiss-madetech) [reply src/app/features/notifications/notification-delink-to-parent/notification-delink-to-parent.component.html:4]: workplace doesn't exist on the component JS ;(

File: src/app/shared/components/cancel-data-owner-dialog/cancel-data-owner-dialog.component.html (hunk 2 above)
Comment (chadtomkiss-madetech) [reply src/app/shared/components/cancel-data-owner-dialog/cancel-data-owner-dialog.component.html:3]: There is no form on this component, it just uses JS functions on each of the buttons on L20-21

**Pull request merged**